ckanext/dbquery/templates/package/read_base.html

```diff
@@ -0,0 +1,18 @@
+{% ckan_extends %}
+
+{% block secondary_content %}
+  {{ super() }}
+  
+  {% if h.check_access('sysadmin') %}
+    <div class="module module-narrow module-shallow">
+      <h2 class="module-heading"><i class="fa fa-database"></i> {{ _('Query History') }}</h2>
+      <div class="module-content">
+        <p>
+          <a class="btn btn-primary" href="{{ h.url_for('dbquery.history') }}">
+            <i class="fa fa-list"></i> {{ _('Show executed queries') }}
+          </a>
+        </p>
+      </div>
+    </div>
+  {% endif %}
+{% endblock %}
```

Comment:
```suggestion
```
It's important to add a queries list link but the read_base for packages is not the rigth place
Show this link in the left bar but for the `dbquery` page (ckanext/dbquery/templates/dbquery/index.html). 
Not here

Reply:
https://github.com/unckan/ckanext-dbquery/pull/9/commits/4fb31cd82795646caf63b513f0db00690927e53d

Reply:
Almost perfect, the `ckanext/dbquery/templates/package/read_base.html` file is not useful now. We are not doing anything in there.
Please, delete it

Reply:
> Almost perfect, the `ckanext/dbquery/templates/package/read_base.html` file is not useful now. We are not doing anything in there. Please, delete it

https://github.com/unckan/ckanext-dbquery/pull/9/commits/db82885c95a9c0b06d674c49e12a169b19540d1e